test/lib/usd/translators/CMakeLists.txt

```diff
@@ -231,6 +231,23 @@ if (MAYA_APP_VERSION VERSION_GREATER_EQUAL 2020)
     endforeach()
 endif()
 
+# Tests using MaterialX, which was introduced in Maya 2022.1 with USD > 21.05.
+if (MAYA_APP_VERSION VERSION_GREATER 2022 AND PXR_VERSION VERSION_GREATER 2102)
```

Comment:
The import/export code does not require a recent USD since we write directly in UsdShade the expected MaterialX graph that represents the Maya nodes we support.
Using 2102 allows the unit test to be run by some of our preflight configurations even though the Vp2 render delegate would be incapable of rendering the results. This is why the unit test is in `test/lib/usd/translators` instead of `test\lib\mayaUsd\render\vp2RenderDelegate`.
I will clarify the comment then to separate the export/import, which can be done in 21.02, from the rendering part, which absolutely requires >21.05.